src/Cryptol/ModuleSystem/Env.hs

```diff
@@ -304,6 +303,12 @@ modContextOf mname me =
        , mctxNames    = localNames
        , mctxNameDisp = R.toNameDisp localNames
        }
+  where getPublicDecls :: LoadedModule -> IfaceDecls
```

Comment:
Just some stylistic things that might be nice to change:
  * Avoid the reverse indentation in `do` blocks 
  * Don't mix composition and `$`. So instead of `f . g . h $ x` I generally tend to write `f (g (h x))`, or sometimes if things get too nested or I am trying to line up things for readability `f $ g $ h x`.